src/MeetingsSDKAdapter.js

```diff
@@ -1,9 +1,14 @@
 import {MeetingsAdapter, MeetingControlState} from '@webex/component-adapter-interfaces';
 import {concat, from, fromEvent, Observable} from 'rxjs';
-import {filter, finalize, map, publishReplay, refCount} from 'rxjs/operators';
+import {filter, finalize, map, merge, publishReplay, refCount} from 'rxjs/operators';
 
 const EVENT_MEDIA_READY = 'media:ready';
+const EVENT_MEDIA_LOCAL_MUTE = 'media:mute';
```

Comment:
Since this is a custom event just for our adapter, I'd prefer for this to have a name that wouldn't have the possibility to collide with any sdk updates. Could we do something like this instead:
```suggestion
const EVENT_MEDIA_LOCAL_MUTE = 'sdkadapter:media:localmute';
```

Then, if we add that functionality to the sdk, we can make a PR to use the new event and remove our custom one.

Reply:
adapter:media:local:mute